.devcontainer/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM mcr.microsoft.com/devcontainers/cpp:1-ubuntu-22.04
```

Comment:
22.04 is pretty old.

according https://mcr.microsoft.com/en-us/artifact/mar/devcontainers/cpp/about

I should probably use 

    mcr.microsoft.com/devcontainers/cpp:ubuntu

Reply:
@s-celles did you make the changes? I'm still getting the same error.
![Screenshot 2024-12-02 at 11 32 51](https://github.com/user-attachments/assets/14e77f77-b2c6-4f82-ade2-4ff09fbb2d51)

Reply:
I'm currently quite busy and candle handle this this week. Sorry

Reply:
@s-celles any updates?

Reply:
Should be ready